bedrock/mozorg/templates/mozorg/about/policy/transparency/includes/reports-nav.html

```diff
@@ -10,7 +10,7 @@
     {# This link should point to the latest report. Past reports are added to 'includes/past-reports.html' #}
     {# There are also links in the faq that need to be updated to point to the latest report. #}
```

Comment:
Note for reviewer: I could not find any FAQ link that needed updating, but a double-check would be good just in case I missed something.

Reply:
I think there were some links originally that must have been removed in a later edit. We should delete this comment to avoid future confusion.